master/internal/checkpoints/postgres_checkpoints_intg_test.go

```diff
@@ -378,75 +378,21 @@ func TestDeleteCheckpoints(t *testing.T) {
 	ckpt1 := uuid.New()
 	checkpoint1 := db.MockModelCheckpoint(ckpt1, allocation)
 	err := db.AddCheckpointMetadata(ctx, &checkpoint1, tr.ID)
-	require.NoError(t, err)
 
 	ckpt2 := uuid.New()
 	checkpoint2 := db.MockModelCheckpoint(ckpt2, allocation)
 	err = db.AddCheckpointMetadata(ctx, &checkpoint2, tr.ID)
```

Comment:
in case the above is done,
```suggestion
	err := db.AddCheckpointMetadata(ctx, &checkpoint2, tr.ID)
```